spec/bat/stemcell_spec.rb

```diff
@@ -88,6 +88,9 @@
       'bosh-warden-boshlite-ubuntu-trusty-go_agent' => false,
       'bosh-warden-boshlite-centos-go_agent' => false,
 
+      # Azure CPI does not support network reconfig
```

Comment:
As you said, we do not implement configure_networks. Next, we will add the support.